src/simmate/apps/badelf/__init__.py

```diff
@@ -1,11 +1 @@
 # -*- coding: utf-8 -*-
-
-# ensure badelf's optional deps are present
-try:
-    import pybader
-    import pyrho
-except:
-    raise Exception(
-        "This app requires two optional dependencies: mp-pyrho and pybader. "
-        "Install these with `conda install -c conda-forge mp-pyrho pybader`"
-    )
```

Comment:
I do like having these up front. This method is preferred, rather than hiding the dependencies away within the methods. My other comment about moving the `toolkit.Grid` into the bader app + bringing this back would be my preference